tests/e2e/test_e2e_indexing_searching.py

```diff
@@ -63,9 +63,11 @@ def test_search():
         "Directed by Isao Takahata, with whom Miyazaki would continue to collaborate for the remainder of his career"  # noqa
         in all_results[1][1]["content"]
     )
+    actual = all_results[1][2]["content"]
     assert (
-        "Specific works that have influenced Miyazaki include Animal Farm (1945)"  # noqa
-        in all_results[1][2]["content"]
+        "Specific works that have influenced Miyazaki include Animal Farm (1945)"
+        in actual
+        or "She met with Suzuki" in actual
```

Comment:
Nearly all looks good to me -- but just a question here, is there a specific reason to include `She met with Suzuki` here? The data processing looks like it still functions the same, so the initial test should still work unless I'm missing something

Reply:
The original test didn't work with me. I'm not sure what was is different on my machine but the 3rd and 4th results come in different order. And like the commit message says, they seem to have the same score.

Reply:
This is strange... 🤔 

The output I've been getting (same locally on multiple machines and in the CI is):

```python
 {'content': "Specific works that have[...]",
  'score': 24.423484802246094,
[...]},
 {'content': 'She met with Suzuki in August 2005, [...]',
  'score': 24.417688369750977,
[...]},
```

The score are extremely similar to the point where it's possibly due to hardware differences, but still kind of odd!

Reply:
(when casting to half precision, they do both have the same score, and since that's what a lot of people do, I'd say the issue is fairly minor and can be blamed on different hardware configs)